File: src/main/java/workshop/userservice/UserService.java
package workshop.userservice;

public class UserService {

    public UserCriminalRecord vettUser(Integer loginId) {
        if (loginId < 100000) {
            return UserCriminalRecord.GOOD;
        } else {
            return UserCriminalRecord.EVIL;
        }
    }

}

File: src/main/kotlin/workshop/common/userservice/UserService.kt
package workshop.common.userservice

class UserService {

    fun vettUser(loginId: Int?): UserCriminalRecord {
        return if (loginId!! < 100000) {
            UserCriminalRecord.GOOD
        } else {
            UserCriminalRecord.EVIL
        }
    }

}
